src/main/resources/public/ts/controllers/controller.ts

```diff
@@ -742,22 +708,8 @@ export const calendarController =  ng.controller('CalendarController',
 
     };
 
-    /**
-     * Check if selected calendar is default calendar
-     * if so return false to disable 'delete' button
-     */
-    $scope.canBeDeleted = function () {
-        if ($scope.calendar && $scope.calendar.is_default) {
-            return false;
-        }
-        return true;
-    }
 
     $scope.removeCalendar = async () => {
-        if ($scope.showButtonsCalendar._id == $scope.calendar._id) {
```

Comment:
showButtonsCalendar and showToggleButtons become undefined automatically, tested with no errors locally.